src/wasm32_bindgen.rs

```diff
@@ -29,33 +26,34 @@ enum RngSource {
     Browser(BrowserCrypto),
 }
 
-thread_local!(
-    static RNG_SOURCE: RefCell<Option<RngSource>> = RefCell::new(None);
-);
+unsafe impl Send for RngSource {}
```

Comment:
Is this safe?

Reply:
Per https://github.com/rustwasm/wasm-bindgen/pull/955, it is not. I've gone back to using `thread_local` for `wasm32_bindgen`. This is perfectly reasonable, as unlike all the other targets, wasm-bindgen actually requires per-thread initialization.